pkg/bpf/bpf_linux.go

```diff
@@ -113,7 +115,29 @@ func UpdateElementFromPointers(fd int, structPtr unsafe.Pointer, sizeOfStruct ui
 	}
 
 	if ret != 0 || err != 0 {
-		return fmt.Errorf("Unable to update element for map with file descriptor %d: %s", fd, err)
+
+		var errMsg error
+
+		switch err {
+		case unix.E2BIG:
+			if strings.Contains(mapName, "lb") {
+				errMsg = errors.New("the bpf LB map is full, please consider resizing it with the flag \"bpf-lb-max-entries\". THIS WILL INTERFERE WITH ALL EXISTING CONNECTIONS")
+			} else {
+				errMsg = errors.New("the bpf map is full, please consider resizing it")
+			}
+		case unix.EEXIST:
+			errMsg = errors.New("Cannot add the element to bpf map, please check the key of the element")
```

Comment:
Not sure if the user can check actually check the key.

How about we rephrase to: "failed to update bpf map %s: specified key already exists"

With the `%s` being the map name.

Note also: https://github.com/golang/go/wiki/CodeReviewComments#error-strings